### PR TITLE
[FIX] fs_image: remove rawCacheKey invalidation

### PR DESCRIPTION
**Pull request by sbejaoui**
Fix view crash when uploading an image

The rawCacheKey is appropriately managed by the base class and reflects the
record's last update datetime (write_date).
Since it lacks a setter, attempting to invalidate its value results in a view crash.
Nevertheless, the value will automatically be updated upon saving the record.

### CODE REVIEW

**Comment by lmignon** [comment]
@sbejaoui  can you add a fragment to include into the addon history... as it's done here https://github.com/OCA/storage/pull/304/files#diff-0c0256bf7a5f02140833fcc325442a42553aac6aaf8b013258f347d6ce2d7e8c

File: fs_image/static/src/views/fields/fsimage_field.esm.js [hunk 1]
```diff
@@ -50,9 +50,6 @@ export class FSImageField extends ImageField {
                     fileTypeMagicWordMap[this.props.value.content[0]] || "png";
                 return `data:image/${magic};base64,${this.props.value.content}`;
             }
-            if (!this.rawCacheKey) {
```
Comment (lmignon) [inline fs_image/static/src/views/fields/fsimage_field.esm.js:53]: @sbejaoui In both cases we redefine a method and don't call super. We should therefore process the 'rawCacheKey'

Comment (lmignon) [reply fs_image/static/src/views/fields/fsimage_field.esm.js:53]: OK my Odoo was not up to date. Indeed, the 'rawCacheKey' is now a property

**Comment by sbejaoui** [comment]
> @sbejaoui can you add a fragment to include into the addon history... as it's done here https://github.com/OCA/storage/pull/304/files#diff-0c0256bf7a5f02140833fcc325442a42553aac6aaf8b013258f347d6ce2d7e8c

done

File: fs_image/readme/newsfragments/305.bugfix [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Remove rawCacheKey invalidation.
```
Comment (lmignon) [inline fs_image/readme/newsfragments/305.bugfix]: @sbejaoui Thank you for this changelog.  A little question. As a non developer or without having done this change, do you think this change can help to know if you need to update your addon? IMO, the title should explain for the end user what's this PR fixes.

**Review by lmignon — CHANGES_REQUESTED** [review]

**Review by lmignon — APPROVED** [review]

**Pull request merged**